torch_xla/csrc/aten_tensor_ops.cpp

```diff
@@ -0,0 +1,105 @@
+#include "torch_xla/csrc/aten_tensor_ops.h"
+
+namespace aten_tensor_ops {
```

Comment:
One empty line after namespace. Did clang-format OK this?
Same in the .h file ...

Reply:
Yea clang-format is fine with it :P

Reply:
But please add, since that is the convention we use in all other files 😉

Reply:
sure :P